src/participant.rs

```diff
@@ -35,6 +34,7 @@ const SIGNATURE_LEN: usize = Signature::BYTE_SIZE;
 pub const IDENTITY_LEN: usize =
     VERSION_LEN + VERIFICATION_KEY_LEN + ENCRYPTION_KEY_LEN + SIGNATURE_LEN;
 
+pub type Signature = ed25519_dalek::Signature;
```

Comment:
I think you'll also need to export `SignatureError` if you want downstream code to be able to do anything with the errors

Reply:
Thanks good catch.